automation/test.sh

```diff
@@ -33,6 +33,31 @@ readonly ARTIFACTS_PATH="${ARTIFACTS-$WORKSPACE/exported-artifacts}"
 readonly TEMPLATES_SERVER="https://templates.ovirt.org/kubevirt/"
 readonly BAZEL_CACHE="${BAZEL_CACHE:-http://bazel-cache.kubevirt-prow.svc.cluster.local:8080/kubevirt.io/kubevirt}"
 
+# KUBEVIRT_LANE_FOCUS allows running only selected lanes.
+# Usage: Add lane names, seperated by space.
+# All the other lanes will fail immediately, saving CI resources,
+# but still guarding from getting the PR merged, until the line
+# is restored to an empty string.
+export KUBEVIRT_LANE_FOCUS=""
```

Comment:
Could you add a real example to the docs? Apart from that looks good to me.

Reply:
done, thx